### DISCUSSION TITLE
Issues with Fan.Control

### DISCUSSION THREAD

**Discussion by 4e576rt8uh9ij9okp** [Rem0o/FanControl.Releases#1715] [General]
Hello there,
so lately I've noticed that Fan Control is becoming very annoying on my system.
Because of that I will soon go with a fan controller because FanControl is very annoying.

1. It broke my saves multiple times.
2. It never starts minimized when I start my PC.
3. Now for some reason it has some issues with initializing sensors.
4. Sometimes on restart FanControl has no effect.
5. It keeps asking me to update even when I see no reason to update, everything works how I want it and I'm happy. But instead it gets prompted each damn time and just annoying me.
6. For some reason it now asks me each time I close the app if I want to save the current settings (while I haven't changed anything), I click on discard and I reopen the window from FanControl from the Tray menu and it still asks me to save or discard it.

![image](https://user-images.githubusercontent.com/104687522/231806737-e3c6e7f4-651e-4085-a64f-56ff1ef42fc0.png)

**Comment by Rem0o** [comment]
Most of the issues you mention are linked to issue number 3, which prompt the dialog you posted as screenshot, except the update thing, which you can dismiss with the checkbox of that dialog. Also, upon updates, some new stuff may be added in into your config file, which triggers a save prompt. 

When sensors are missing, it will unminimize the program to get your attention since it won't be working properly in the background.  Is it always the Nvidia stuff that goes missing? What kind of  a setup do you got?
